tests/modules/test_lint.py

```diff
@@ -505,6 +505,86 @@ def test_modules_environment_yml_file_not_array(self):
         assert len(module_lint.warned) >= 0
         assert module_lint.failed[0].lint_test == "environment_yml_valid"
 
+    def test_modules_environment_yml_file_mixed_dependencies(self):
+        """Test linting a module with mixed-type dependencies (strings and pip dict)"""
+        with open(
+            Path(
+                self.nfcore_modules,
+                "modules",
+                "nf-core",
+                "bpipe",
+                "test",
+                "environment.yml",
```

Comment:
could we add the path to a variable? it is difficult to read since it's split into many lines, and used more than one

Reply:
was thinking the same, will do.